src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.Linux.cs

```diff
@@ -19,6 +20,15 @@ public class LibGpiodDriver : UnixDriver
 
         protected internal override int PinCount => Interop.libgpiod.gpiod_chip_num_lines(_chip);
 
+        public enum RequestFlag : ulong {
```

Comment:
This should probably be internal or private.

Reply:
Thanks for the review. I fix it on d423507